### PR TITLE
Add to bundle

### PR DESCRIPTION
**Pull request by markkramerus**
Added remaining elements to bundle: Labs, vitals, other

### CODE REVIEW

File: input/fsh/Bundle.fsh [hunk 1]
```diff
@@ -21,14 +21,51 @@ Description: "A collection of data for an mCODE cancer patient."
...
+    coreLaboratory 0..* MS and
+    otherResources 0..1
+
+* entry[cancerPatient] ^short = "Cancer Patient"
+* entry[primaryCancerCondition] ^short = "Primary Cancer Condition"
+* entry[secondaryCancerCondition] ^short = "Secondary Cancer Condition(s)"
+* entry[cancerDiseaseStatus] ^short = "Cancer Disease Status(es)"
+* entry[performanceStatus] ^short = "Performance Status(es) (ECOG or Karnofsky)"
+* entry[tumorMarker] ^short = "Tumor Marker Tests"
+* entry[cancerRelatedMedicationRequest] ^short = "Cancer-RelatedMedicationRequest(s)"
```
Comment (masnick) [inline input/fsh/Bundle.fsh:40]:
```suggestion
* entry[cancerRelatedMedicationRequest] ^short = "Cancer-Related MedicationRequest(s)"
```

File: input/fsh/Bundle.fsh [hunk 2]
```diff
@@ -21,14 +21,51 @@ Description: "A collection of data for an mCODE cancer patient."
...
+* entry[cancerRelatedProcedure] ^short = "Cancer-Related Procedure(s)"
+* entry[cancerStage] ^short = "Cancer Stage Data"
+* entry[cancerGeneticVariant] ^short = "Cancer Genetic Variant(s)"
+* entry[cancerGenomicsReport] ^short = "Cancer Genomics Report(s)"
+* entry[geneticSpecimen] ^short = "Genetic Specimen(s)"
+* entry[genomicRegionStudied] ^short = "Genomic Region(s) Studied"
+* entry[cancerRelatedComorbidities] ^short = "Cancer-Related Comorbities"
+* entry[vitalSign] ^short = "Vital Sign"
```
Comment (masnick) [inline input/fsh/Bundle.fsh:48]:
```suggestion
* entry[vitalSign] ^short = "Vital Sign(s)"
```

File: input/fsh/Bundle.fsh [hunk 3]
```diff
@@ -21,14 +21,51 @@ Description: "A collection of data for an mCODE cancer patient."
...
+* entry[coreLaboratory] ^short = "CBC and CMP Laboratory Results"
```
Comment (masnick) [inline input/fsh/Bundle.fsh:49]: Do we define what "core" means? And are we really limiting to CBC and CMP, or should this say "for example, CBC and CMP..."?

File: input/fsh/ExtendedExample1.fsh [hunk 4]
```diff
@@ -681,3 +681,27 @@ Description: "Extended example 1: example showing DCIS diagnosis"
...
+Description: "Extended example 1: neutrophils lab test"
+* status = #final "final"
+* category = ObsCat#laboratory "Laboratory"
+* code = LNC#770-8 "Neutrophils/100 leukocytes in Blood by Automated count"
+* subject = Reference(example1-mcode-cancer-patient)
+* effectiveDateTime = "2018-03-06T00:00:00Z"
+* valueQuantity = 40.0 '%'
+* referenceRange.low = 37.0 '%'
+* referenceRange.high = 67.0 '%'
+* referenceRange.appliesTo = RefMeaning#normal "Normal Range"
```
Comment (masnick) [inline input/fsh/ExtendedExample1.fsh:707]: We need to update the prose version of the "extended example" (now called "scenario 1" in #30). I recommend we merge this in first, then merge in #30, and then update the prose to reduce merge/filename conflicts.

File: input/fsh/Bundle.fsh [hunk 5]
```diff
@@ -43,6 +80,26 @@ Description: "A collection of data for an mCODE cancer patient."
...
+Id: mcode-core-laboratory
+Description: "Abstract profile for discrimination of the core laboratory slice in the mCODE Patient Bundle."
+* ^abstract = true
+* code from CoreLaboratoryVS (required)
+
+Profile: OtherResources
+Parent: Bundle
+Id: mcode-other-resources
+Description: "Abstract profile for to define the 'other resources' bundle in the mcode patient bundle."
+* ^abstract = true
```
Comment (masnick) [inline input/fsh/Bundle.fsh:100]: Do we need an example for this? (Not necessarily for this PR but in general.)

**Comment by masnick** [comment]
We should merge `consistent-use-of-category` into master **before** we merge this in, or #38 may get confusing.

File: input/fsh/Bundle.fsh (hunk 1 above)
Comment (markkramerus) [reply input/fsh/Bundle.fsh:40]: Actually should be Cancer-Related Medication Request(s). Fixed.

File: input/fsh/Bundle.fsh (hunk 3 above)
Comment (markkramerus) [reply input/fsh/Bundle.fsh:49]: Core means CBC and CMP with specific LOINC codes. I haven't found a way to loosen this and still have the slicing work.

Comment (markkramerus) [reply input/fsh/Bundle.fsh:49]: We can stick with this until we find a better approach.

File: input/fsh/ExtendedExample1.fsh (hunk 4 above)
Comment (markkramerus) [reply input/fsh/ExtendedExample1.fsh:707]: Sure

File: input/fsh/Bundle.fsh (hunk 5 above)
Comment (markkramerus) [reply input/fsh/Bundle.fsh:100]: I'm adding one now.

Comment (markkramerus) [reply input/fsh/Bundle.fsh:100]: Pushing...

**Pull request merged**